### PR TITLE
Save some space due to shorter MMU error texts

### PR DESCRIPTION
**Pull request by DRracer**
applies https://github.com/prusa3d/Prusa-Error-Codes/pull/62

PFW-1401

### CODE REVIEW

File: Firmware/mmu2/errors_list.h [hunk 1]
```diff
@@ -192,41 +192,44 @@ static const char * const errorTitles [] PROGMEM = {
...
-static const char descSELECTOR_TMC_DRIVER_SHORTED[] PROGMEM_I1 = ISTR("Short circuit on the Selector TMC driver. Check the wiring and connectors. If the issue persists contact support.");
-static const char descIDLER_TMC_DRIVER_SHORTED[] PROGMEM_I1 = ISTR("Short circuit on the Idler TMC driver. Check the wiring and connectors. If the issue persists contact support.");
-static const char descMMU_NOT_RESPONDING[] PROGMEM_I1 = ISTR("MMU unit not responding. Check the wiring and connectors. If the issue persists contact support.");
-static const char descCOMMUNICATION_ERROR[] PROGMEM_I1 = ISTR("MMU unit not responding correctly. Check the wiring and connectors. If the issue persists contact support.");
+static const char descSELECTOR_CANNOT_HOME[] PROGMEM_I1 = ISTR("The Selector cannot home properly. Check for anything blocking its movement.");
+static const char descCANNOT_MOVE[] PROGMEM_I1 = ISTR(""); // these errors have been disabled
+//static const char descSELECTOR_CANNOT_MOVE[] PROGMEM_I1 = ISTR("The Selector cannot move. Check for anything blocking its movement. Check the wiring is correct.");
```
Comment (3d-gussner) [inline Firmware/mmu2/errors_list.h:204]: Is it active or not see https://github.com/prusa3d/Prusa-Error-Codes/commit/4515a478384d380f24446be4cf000ae046fabd3d#diff-fd4c0f7e47a1cbb49a66cbb4936bae0c72484f96809baa5d1eb322b2e4f957ccR69

File: Firmware/mmu2/errors_list.h [hunk 2]
```diff
@@ -192,41 +192,44 @@ static const char * const errorTitles [] PROGMEM = {
...
+static const char descIDLER_CANNOT_HOME[] PROGMEM_I1 = ISTR("The Idler cannot home properly. Check for anything blocking its movement.");
+//static const char descIDLER_CANNOT_MOVE[] PROGMEM_I1 = ISTR("The Idler cannot move properly. Check for anything blocking its movement. Check the wiring is correct.");
```
Comment (3d-gussner) [inline Firmware/mmu2/errors_list.h:206]: see https://github.com/prusa3d/Prusa-Error-Codes/commit/4515a478384d380f24446be4cf000ae046fabd3d#diff-fd4c0f7e47a1cbb49a66cbb4936bae0c72484f96809baa5d1eb322b2e4f957ccR83

**Review by 3d-gussner — COMMENTED** [review]
Please see comments

File: Firmware/mmu2/errors_list.h (hunk 1 above)
Comment (DRracer) [reply Firmware/mmu2/errors_list.h:204]: for now let's keep the generic "move" text there

File: Firmware/mmu2/errors_list.h (hunk 2 above)
Comment (DRracer) [reply Firmware/mmu2/errors_list.h:206]: dtto. keep the "move" text there but empty, the MMU doesn't send it now

**Review by 3d-gussner — APPROVED** [review]

**Pull request merged**